### DISCUSSION TITLE
Sponsorship

### DISCUSSION THREAD

**Discussion by subsetpark** [janet-lang/janet#676] [Q&A] [answered]
Have you ever considered setting up a sponsorship model for Janet? I wouldn’t be surprised if a few of us were happy to do some kind of Patreon-style sponsorship, monthly amount, et cetera. 

Of course I suppose it would require some desire to either work on Janet professionally or to pay others to work on Janet, which may or may not be in your long term plan.

**Comment by pyrmont** [comment, marked as answer]
You can [sponsor](https://github.com/sponsors/bakpakin) @bakpakin through GitHub. There's a few people doing that at the moment :)

**Reply by subsetpark** [reply]
Fantastic! Maybe we should just put a link to that page in the janet README? Unless I missed it?

**Reply by pyrmont** [reply]
Sounds like a good idea to me!

**Reply by uvtc** [reply]
Created #677 for this.
